.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,50 +1,36 @@
+<!--- Type an x in the box [x] before submitting --->
 
-### Pull Request Checklist
+### Submission Checklist
 
-* [ ] Read and followed this repo's [Contributing Guidelines](https://github.com/spyder-ide/spyder/blob/master/CONTRIBUTING.md)
-* [ ] Based this PR on the latest version of the correct branch (master or 3.x)
-* [ ] Followed [PEP 8](https://www.python.org/dev/peps/pep-0008/) code style
-* [ ] Ensured this pull request hasn't eliminated unrelated blank lines/spaces,
-      modified the ``spyder/defaults`` directory, or added new icons/assets
-* [ ] Wrote at least one-line docstrings following PEP 257for any new functions
-* [ ] Added at least one unit test covering the changes, if at all possible
-* [ ] Described the changes and the motivation for them below
-* [ ] Noted what issue(s) this pull request resolves, creating one if needed
-* [ ] Included a screenshot, if this PR makes any visible changes to the UI
+<!--- I have read the Contributing Guidelines:                         --->
+<!--- https://github.com/spyder-ide/spyder/blob/master/CONTRIBUTING.md --->
+<!--- and followed PEP 8, PEP 257 and Spyder's code style:             --->
+<!--- https://github.com/spyder-ide/spyder/wiki/Dev:-Coding-Style      --->
 
+By submitting this Pull Request or typing my (user)name below,
+I affirm the [Developer Certificate of Origin](https://developercertificate.org)
+with respect to all commits and content included in this PR,
+and understand I am releasing the same under Spyder's MIT (Expat) license.
 
-### Developer Certificate of Origin Affirmation
-
-By submitting this Pull Request or typing my name below, I affirm the
-[Developer Certificate of Origin](https://developercertificate.org/)
-with respect to both the content of the contribution itself and this post,
-and understand I am releasing it under Spyder's MIT (Expat) license.
-
-
-<!--- TYPE YOUR GITHUB USERNAME AFTER THE FOLLOWING STATEMENT ---!>
+<!--- TYPE YOUR USER/NAME AFTER THE FOLLOWING: --->
 I certify the above statement is true and correct:
 
-<!--- Note that you (not Spyder) retain copyright ownership of your work, --->
-<!--- and may license it to other parties under the terms of your choice. --->
-<!--- Contact us before incorporating any content from external projects. --->
-
 
 ## Description of Changes
 
-<!--- Explain what you've done and why. --->
+* [ ] Wrote at least one-line docstrings (for any new functions)
+* [ ] Added unit test(s) covering the changes (if testable)
+* [ ] Included a screenshot (if affecting the UI)
+
+<!--- Explain what you've done and why --->
 
 
 
 
 ### Issue(s) Resolved
 
-<!--- Pull requests should typically resolve one, and preferably only one --->
-<!--- outstanding issue; create a new one if no relevant issue exists. --->
-<!--- List the issue(s) below, in the form "Fixes #1234"; one per line. --->
+<!--- List the issue(s) below, in the form "Fixes #1234"; one per line --->
```

Comment:
This is great, I like this.

The info that you deleted rightfully here can still be very valuable for first time contributors. We should add this in the contributing guidelines if it is not already in and also in a "Welcome" message that we can rig with Probot later for first time contributors.

Reply:
Yeah, I've added a few things to the Spyder code style page but there's still a lot of things we don't explicitly explain, from e.g. putting ``PR:`` on PR titles to our general project standards (encoding, line ending, file header, etc) to our various patterns and idiomatic conventions (e.g. platform detection, accessing config data, etc). I was going to add a section summarizing that to the contributing guide at some point soon, and we can list all the specifics that we know, adding more as they come up, to the Coding style wiki page, starting with these.

Reply:
OK thanks!